docs/modules/ROOT/pages/index.adoc

```diff
@@ -664,11 +664,40 @@ The priority is to be set based on the priorities of the existing decorators whi
 .ProcessorDecoratorPriorities.java
 [source,java]
 ----
-include::../../../../api/src/main/java/io/quarkiverse/kafkastreamsprocessor/api/decorator/processor/ProcessorDecoratorPriorities.java[tag=priorities]
+     /**
```

Comment:
A plain copy is guaranteed to no longer be in line with the code in the long run.
Might be worth digging further to understand if we could make the inclusion work in some way.

If not, the `priorities` tag must also be removed from the target code.

Reply:
Conclusion: some repos like quarkus-cxf use include directives from tags in source code, but they duplicate the code in subfolders of the doc root to have it accessible when Antora aggregates the doc for all repos.
This is overkill for this repo, we'll stick to plain text copy in the .adoc file.